pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -321,10 +324,23 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 
 	// nodes in clusters WITHOUT multi-networking are expected to have only 1 network-interface with 1 alias IP range.
 	if len(instance.NetworkInterfaces) == 1 && len(instance.NetworkInterfaces[0].AliasIpRanges) == 1 {
```

Comment:
I had to modify the other ifs to check for ipv6 only networks too https://github.com/kubernetes/cloud-provider-gcp/commit/ce1a81aeb805a9d69562c8f0bde819f459bb182f

Reply:
Done